testing/pypy/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor:Mihnea Saracin <mihnea.saracin@rinftech.com>
+# Maintainer:Mihnea Saracin <mihnea.saracin@rinftech.com>
+pkgname=pypy
+pkgver=5.7.1
+pkgrel=0
+pkgdesc="pypy2"
+url="https://bitbucket.org/pypy"
+arch="all"
+license="MIT"
+depends="libssl1.0 libcrypto1.0 libffi libbz2 "
+makedepends="gcc make python linux-headers  libffi-dev pkgconf zlib-dev bzip2-dev 
+            sqlite-dev ncurses-dev readline-dev openssl-dev expat-dev openssl-dev gdbm-dev tk tk-dev libc-dev py-cffi paxmark"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="https://bitbucket.org/$pkgname/$pkgname/downloads/pypy2-v$pkgver-src.tar.bz2
+        alpine.patch"
+builddir="$srcdir/pypy2-v$pkgver-src/pypy/goal"
+
+
+prepare(){
```

Comment:
This entire function is part of the default prepare implementation and can be omitted.